pkg/api/validation/validation.go

```diff
@@ -738,7 +747,7 @@ func ValidateMinionUpdate(oldMinion *api.Node, minion *api.Node) errs.Validation
 // Refer to docs/resources.md for more details.
 func validateResourceName(str string) errs.ValidationErrorList {
 	if !util.IsQualifiedName(str) {
-		return errs.ValidationErrorList{fmt.Errorf("invalid compute resource typename format %q", str)}
+		return errs.ValidationErrorList{fmt.Errorf("invalid compute resource typename %q which %s", str, QualifiedNameErrorMsg)}
```

Comment:
please return errs.NewFieldInvalid(...) here - see everywhere else for examples.  This should never have been this way :)